### DISCUSSION TITLE
Calibration documentation

### DISCUSSION THREAD

**Discussion by b-goldsmith** [unitaryfund/mitiq#2498] [General]
Since I have spending a lot of time with the `Calibrator` module, I started a list of things that should be addressed. While normally I would add these in my Notes app and then forget about them, I am trying to be a bit more responsible. I am hoping that we can discuss these and create issues out of them so that they can be addressed.

1. Currently, the docs on Calibration do not mention the the `log` parameter as part of the `run` method. I think this feature is very useful and should be highlighted in the docs.
![image](https://github.com/user-attachments/assets/d90f54a2-d55f-41ac-81c1-7fb4d9204f63)

3. Also, the values of `log` (flat and cartesian) are not included in the documentation (including the [API-doc](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.calibration.calibrator.Calibrator.run)), let alone the difference between them. 
4. The API-docs do include [log_results_cartesian()](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.calibration.calibrator.ExperimentResults.log_results_cartesian) and [log_results_flat()](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.calibration.calibrator.ExperimentResults.log_results_flat) though the example output is mangled and not a good representation. In the actual source code, the example output is perfect for both.
![image](https://github.com/user-attachments/assets/0420cb19-e233-429e-8e75-109ebe4c4b2d)
![image](https://github.com/user-attachments/assets/ff626b85-5ad6-47c6-b2cc-e4dc259430ac)

**Comment by b-goldsmith** [comment]
The [ZNE Calibration with Qiskit](https://mitiq.readthedocs.io/en/stable/examples/calibration-tutorial.html) does use `cal.run(log="flat"`, including some explanation about the results.

**Reply by natestemen** [reply]
Thanks a bunch for these Brian! Super useful feedback. In fact maybe the individual points should just be issues!? Would you mind opening some (as you see fit)? If you don't have time, I can too. Just let me know.

**Reply by b-goldsmith** [reply]
Thanks, Nate. Now that things have calmed down I will go ahead and open up issues for each. I will also tackle them too, unless you want to leave them as good first time issues for newcomers.

**Reply by natestemen** [reply]
If you want to take em, please do!

**Reply by b-goldsmith** [reply]
I have created the issues. I am cool if we want to close this discussion.

**Reply by natestemen** [reply]
Thank you Brian!
